cli/babeltrace.c

```diff
@@ -620,6 +620,7 @@ int load_dynamic_plugins(struct bt_value *plugin_paths)
 	return ret;
 }
 
+#ifdef BT_BUILT_IN_PLUGINS
```

Comment:
No problem per say, I thought the log message about loading static plugins when they were not built a bit confusing.